src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -24,6 +24,7 @@ export default () => `
 						<li><a href="command:registeredServers.addConnection">${escape(localize('welcomePage.newConnection', "New connection"))}</a></li>
 						<li><a href="command:workbench.action.files.newUntitledFile">${escape(localize('welcomePage.newQuery', "New query"))}</a></li>
 						<li><a href="command:notebook.command.new">${escape(localize('welcomePage.newNotebook', "New notebook"))}</a></li>
+						<li><a href="command:azdata.resource.create">${escape(localize('welcomePage.newResource', "New resource"))}</a></li>
```

Comment:
I thought we were going to come up with a better name for this command.  I don't think "New resource" makes sense in this context.

Reply:
this is just an initial commit for Alan to review the changes I made for adding to the welcome page. We'll make changes to it as we go.

Reply:
I'd recommend against making changes to the welcome screen until we've made a decision about basic things such as the name of the commands.  You could stick something in the command palette if you need a way to invoke the command, but I'd rather not add UI elements that are confusing, particularly if we think they may change.

Reply:
Sure Karl. These changes are on the deployment-workstream branch and not to master, so I took the liberty to make changes just to test if we can access external host content from welcome page. I'll not make any changes until we have made a decision.